environment.yml

```diff
@@ -39,6 +39,7 @@ dependencies:
   - pytest
   - pytest-cov
   - xgboost
+  - polars<1.0
```

Comment:
any particular reason for the pin?

Reply:
New version with some breaking changes just released. https://docs.pola.rs/releases/upgrade/1/ Would like to wait to double check before allowing this new version.